src/e2e/events/crud.spec.ts

```diff
@@ -0,0 +1,151 @@
+    await page.getByRole("button", { name: "Dalej" }).click();
+    await page.getByText("Dodaj współorganizatorów").waitFor();
+    await page.getByRole("button", { name: "Dalej" }).click();
+    await page.getByText("Dodaj atrybuty").waitFor();
+    await page.getByRole("button", { name: "Utwórz wydarzenie" }).click();
+    // Wait for the event to be created
+    await page.waitForURL("/dashboard/events/*");
+    // Verify the event information
+    await expect(page.locator("h1")).toContainText(testData.name);
+    await expect(page.getByRole("main")).toContainText(testData.organizer);
+    await expect(page.getByRole("main")).toContainText("13.08.2025 13:30");
+    await expect(page.getByRole("main")).toContainText("14.08.2025 15:30");
+    await expect(page.getByRole("main")).toContainText(testData.description);
+  });
+
+  await test.step("should edit an event", async () => {
+    await page.getByRole("link", { name: "Edytuj wydarzenie" }).click();
+    await page.waitForURL("/dashboard/events/*/settings");
+    await page
+      .getByRole("textbox", { name: "Nazwa" })
+      .fill(`${testData.name} edited`);
+    await page.getByRole("button", { name: "August 13th," }).click();
+    await page.getByRole("button", { name: "Friday, August 15th," }).click();
+    await page.getByRole("button", { name: "August 14th," }).first().click();
+    await page
+      .getByRole("button", { name: "Saturday, August 16th," })
+      .last()
+      .click();
+    await page
+      .getByRole("textbox", { name: "Opis" })
+      .fill(`${testData.description} edited`);
+    await page
+      .getByRole("textbox", { name: "Miejsce (opcjonalnie)" })
+      .fill(`${testData.location} edited`);
+    await page
+      .getByRole("textbox", { name: "Organizator (opcjonalnie)" })
+      .fill(`${testData.organizer} edited`);
+    await page.getByRole("button", { name: "Zapisz" }).click();
+    // Wait for the event to be updated
+    await page.getByText("Wydarzenie zostało zapisane").waitFor();
+    await page.goBack();
+    // Verify the event information
+    await expect(page.locator("h1")).toContainText(`${testData.name} edited`);
+    await expect(page.getByRole("main")).toContainText(
+      `${testData.organizer} edited`,
+    );
+    await expect(page.getByRole("main")).toContainText("15.08.2025 13:30");
+    await expect(page.getByRole("main")).toContainText("16.08.2025 15:30");
+    await expect(page.getByRole("main")).toContainText(
+      `${testData.description} edited`,
+    );
+  });
+
+  /**
+   * NOTE: This is that one test which needs to be run after the event creation test,
+   * because it checks for the slug that is already used.
+   * The problem is that if this one fails somehow, the deletion test will not be able to run,
+   * and the event will remain in the database, making the next test runs fail.
+   * For now I have no idea how to solve this issue - Maciej
+   */
```

Comment:
opisałem to dokładnie w komentarzu wyżej - odnośnie tego, to ja jestem fanem robienia nowego użytkownika dla każdego testu

a jeśli to nie jest możliwe to dodanie do api specjalnego endpointu to wyczyszczenia użytkownika (a najlepiej usunięcia), tak, że po każdym teście zaczynamy od zera

Reply:
dobrze jest też połączyć oba, żeby nie było syfu w bazie